lesson02/data_sources.tf

```diff
@@ -28,11 +29,12 @@ data "aws_ami" "amazon_linux" {
 }
 
 data "template_file" "deploy_sh" {
-  template = "${file("${path.module}/userdata.sh")}"
+  template = file("${path.module}/userdata.sh")
 
-  vars {
-    docker_tag  = "${var.metadata["appversion"]}"
-    ENVIRONMENT = "${var.env}"
+  vars = {
+    docker_tag  = var.metadata["appversion"]
+    ENVIRONMENT = var.env
     HOSTNAME    = "${var.metadata["appname"]}-${var.env}-ec2-${var.metadata["appversion"]}"
   }
 }
+
```

Comment:
styling: let's remove the extra line

Reply:
we can do this in the next cycle this is not a blocker, thanks for pointing that out

Reply:
This will be done in another PR. Created an issue to keep track of this #33